### PR TITLE
Run GL commands on the GStreamer GL thread

### LINKED ISSUE #25268: GStreamer plugin should run GL code on the GL thread
GL contexts in gstreamer have thread affinity, c.f. https://gstreamer.freedesktop.org/documentation/gl/gstglcontext.html?gi-language=c#gst_gl_context_thread_add

### PR DESCRIPTION
**Pull request by asajeffrey**
GStreamer has thread affinity for GL contexts.

---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors
- [x] These changes fix #25268 
- [x] These changes do not require tests because

### CODE REVIEW

File: ports/gstplugin/servowebsrc.rs [hunk 1]
```diff
@@ -55,8 +55,10 @@ use gstreamer_base::BaseSrcExt;
 use gstreamer_gl::GLContext;
 use gstreamer_gl::GLContextExt;
 use gstreamer_gl::GLContextExtManual;
+use gstreamer_gl_sys::gst_gl_context_thread_add;
```
Comment (Manishearth) [inline ports/gstplugin/servowebsrc.rs]: is there no exported rust function for this? should there be one? we should file an upstream bug

**Review by Manishearth — APPROVED** [review]

File: ports/gstplugin/servowebsrc.rs (hunk 1 above)
Comment (asajeffrey) [reply ports/gstplugin/servowebsrc.rs]: Filed https://gitlab.freedesktop.org/gstreamer/gstreamer-rs/issues/229

**Comment by jdm** [comment]
@bors-servo retry
* #24726

**Comment by jdm** [comment]
@bors-servo retry
* #22212 #24762

**Comment by jdm** [comment]
@bors-servo retry
* #25237

**Pull request merged**